src/catkin_pkg/changelog_generator.py

```diff
@@ -266,6 +267,13 @@ def generate_version_content(log_entries, vcs_client=None, skip_contributors=Fal
     return data
 
 
+def escape_trailing_underscores(line):
+    if line.endswith('_'):
+        line = line[:-1] + '\_'
+    line = re.sub(r'(\w+)_([^\w])', '\\1\\_\\2', line)
```

Comment:
Please add a comment explaining what the regex does, it's obvious now but may not be six months from now (or even a week :-))